index.html

```diff
@@ -47,5 +47,80 @@
         <button class="submit_button" type="button">order now</button>
       </div>
     </header>
+    <main>
+      <section id="hero">
+        <div class="hero_text">
+          <div class="hero_title">
+            <h3>Health DNA test + Wellness Plans</h3>
+            <h1>
+              Get The Only
+              <a
+                href="https://zpe.gov.pl/a/dna---nosnik-informacji-genetycznej/D18yvChME"
+                >DNA</a
+              >
+              Test That Provides Truly Personalized Insights Based on Your
+              Genetic Results
+            </h1>
+          </div>
+
+          <div class="hero_frame">
+            <div class="frame_text">
+              <img src="./public/icons/Tick Square.png" />
+              <p>Analysis of health-related genetic variants</p>
+            </div>
+            <div class="frame_text">
+              <img src="./public/icons/Tick Square.png" />
+              <p>Summary of genetic risks</p>
+            </div>
+            <div class="frame_text">
+              <img src="./public/icons/Tick Square.png" />
+              <p>40+ personalized wellness reports</p>
+            </div>
+            <div class="frame_text">
+              <img src="./public/icons/Tick Square.png" />
+              <p>Custom health pln builder</p>
+            </div>
+          </div>
+
+          <div class="button-price">
```

Comment:
so for naming do you use - or _?? keep consistent !
additionaly if you would like to use BEM metodic it should be
Block_Element-Modifier so
hero_button-price
if you use OOCSS
you should just call it
price

additionaly
why you have so weird nesting construction ?